Kentico/KInspector
# Fix filtering

### Motivation

Makes the filtering in new UI actually work... 😉

### Checklist


### How to test

1. Build UI & solution
1. Connect to an instance
1. Apply filters
1. Observe that filters change the listing
